### PR TITLE
pynicotine: remove "-p" command line argument for Plugins directory

### PR DESCRIPTION
**Pull request by slook**
The plugin directory option is probably redundant now, see discussion https://github.com/nicotine-plus/nicotine-plus/discussions/1834#discussioncomment-2787164 

- Removed: "`-p`" / "`--plugins`" command line option/argument
- Changed: "`-u`" / "`--user-data`" now includes setting the plugins folder location
- Changed: Help string for the -u / --user-data option to reflect the functional change
+ Added: Create an empty `plugins` folder inside the user data folder to help the user find where to put their plugins

@Nachtalb @Inso-m-niaC and others: Please feel free to comment and/or protest about the changes, especially as to the removal of the `-p` command line argument if this might cause any issues :)

This needs to be mentioned in the release notes.

### CODE REVIEW

**Comment by mathiascode** [comment]
Same opinion as the other PR. If we're removing arguments, we should remove them entirely.

**Comment by mathiascode** [comment]
Can you split the new args into a new PR?

**Comment by slook** [comment]
No not immediately, because the new args depend on a new function in this PR. All I can do is revert for now.

I shall create a new PR for the experimental --profile and --portable arguments after this PR has been merged.

File: pynicotine/config.py [hunk 1]
```diff
@@ -73,6 +70,12 @@ def __init__(self):
         self.issue_tracker_url = "https://github.com/nicotine-plus/nicotine-plus/issues"
         self.translations_url = "https://nicotine-plus.org/doc/TRANSLATIONS"
 
+        # Set defaults to use if nothing is specified on command line
+        self.config_dir, self.data_dir = self.get_user_directories()
+        self.plugin_dir = None
```
[the remark this answers is not in the document]
Comment (slook) [reply pynicotine/config.py]: Great idea! Done.

**Comment by slook** [comment]
There is an actions error I do not know how to resolve please help:

https://github.com/nicotine-plus/nicotine-plus/runs/6597779428?check_suite_focus=true

**Comment by mathiascode** [comment]
Implemented in 970863bdb185de8ea79b6906311a1192133c154c

**Pull request closed without merging**